dns/checkip/src/opnsense/mvc/app/views/OPNsense/DynDNS/checkip.volt

```diff
@@ -0,0 +1,219 @@
+          <th data-column-id="name" data-width="6em" data-type="string">{{ lang._('Name') }}</th>
+          <th data-column-id="url" data-width="24em" data-type="string" data-sortable="false">{{ lang._('URL') }}</th>
+          <th data-column-id="verifysslpeer" data-width="9em" data-type="string" data-formatter="boolean" data-css-class="text-center" data-header-css-class="text-center">{{ lang._('Verify SSL Peer') }}</th>
+          <th data-column-id="description" data-width="15em" data-type="string">{{ lang._('Description') }}</th>
+          <th data-column-id="uuid" data-type="string" data-identifier="true" data-visible="false">{{ lang._('ID') }}</th>
+          <th data-column-id="commands" data-width="8em" data-formatter="commands" data-sortable="false">{{ lang._('Commands') }}</th>
+        </tr>
+      </thead>
+      <tbody>
+      </tbody>
+      <tfoot>
+      <tr>
+        <td colspan="6"></td>
+          <td>
+            <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
+            <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button>
+          </td>
+        </tr>
+      </tfoot>
+    </table>
+
+    <table class="table">
+      <tr>
+        <td><a id="help_for_services" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a></td>
+        <td>
+          <div class="hidden" data-for="help_for_services">
+              <p><?= htmlspecialchars(gettext('These services will be used to check IP addresses for Dynamic DNS services, and RFC 2136 entries that have the "Use public IP" option enabled.')) ?></p>
+              <p><?= htmlspecialchars(gettext('If a check IP service with a matching interface and IP version assignment is not found. The one enabled here as the default will be used.')) ?></p>
+              <p><?= htmlspecialchars(gettext('The server must return the client IP address as a string in the format of the "address capture regular expression" option that is specified in the service configuration.')) ?></p>
+              <hr/>
+                 <?= htmlspecialchars(gettext('Examples of address capture regular expression')) ?><br/>
+                 <?= htmlspecialchars(gettext('HTML containing address:')) ?>
+            <pre><?= htmlspecialchars('<body>Current IP Address: (.*)</body>') ?></pre>
+                 <?= htmlspecialchars(gettext('Text only address:')) ?>
+            <pre><?= htmlspecialchars('^(.*)$') ?></pre>
+              <hr/>
+                 <?= htmlspecialchars(gettext('Examples of server PHP')) ?><br/>
+                 <?= htmlspecialchars(gettext('HTML:')) ?>
+            <pre><?= htmlspecialchars('<html><head><title>Current IP Check</title></head><body>Current IP Address: <?=$_SERVER[\'REMOTE_ADDR\']?></body></html>') ?></pre>
```

Comment:
valid HTML needs a doctype btw.

Reply:
For once I can honestly say I don't care about the spec. ;)
When testing it may be viewed in a browser.  But does cURL care?
These are just examples.  People can add doctype if they like.  Even checkip.dyndns.org doesn't supply a doctype.
There is nothing version specific that needs to be rendered...